cmd/kubeadm/app/cmd/reset.go

```diff
@@ -105,7 +105,7 @@ func newResetData(cmd *cobra.Command, opts *resetOptions, in io.Reader, out io.W
 		return nil, err
 	}
 
-	initCfg := &kubeadmapi.InitConfiguration{}
+	var initCfg *kubeadmapi.InitConfiguration
```

Comment:
Add some comment to avoid same wrong refactor?
It would be nice to add some unit tests about this.

Reply:
Done